MikroTik/Current/Site-to-Site_VPN_using_MikroTik_RouterOS.md

```diff
@@ -0,0 +1,60 @@
+This configuration template applies to **MikroTik** physical/virtual appliance running **MikroTik RouterOS 6.40.4** or greater. The configuration is probably applicable to older version of RouterOS, but not validated here. It configures an IPSec VPN tunnel connecting your on-premise VPN appliance with the Azure gateway. Things that begin with "azure-" are variable names and can be changed consistently.
+
+- Vpn Type: **RouteBased**
```

Comment:
Well, no route based ipsec support for mikrotik, only policy-based ipsec is supported.